app/Http/Controllers/SettingsController.php

```diff
@@ -186,11 +187,19 @@ public function postSaveFirstAdmin(SetupUserRequest $request)
             $settings->save();
 
             if (Input::get('email_creds')=='1') {
-                Mail::send(['text' => 'emails.firstadmin'], $data, function ($m) use ($data) {
+                $data = array();
+                $data['email'] = $user->email;
+                $data['username'] = $user->username;
+                $data['first_name'] = $user->first_name;
```

Comment:
Can we add last name in here too? In some cultures, addressing someone by only the first name is considered very rude. (This came up in an issue a month or two back.)

Reply:
No problem, I will update the Welcome notification to include it as well.

Reply:
Awesome, thanks! <3